docs/FAQ.md

```diff
@@ -79,25 +81,35 @@ For more about the closed source parts of VS Code, see [vscodium/vscodium](https
 
 ## How can I request a missing extension?
 
-Please open a new issue and select the `Extension request` template.
+To add an extension to Open-VSX, please see [open-vsx/publish-extensions](https://github.com/open-vsx/publish-extensions).
+We no longer plan to add new extensions to our legacy extension gallery.
 
 If an extension is not available or does not work, you can grab its VSIX from its Github releases or
 build it yourself. Then run the `Extensions: Install from VSIX` command in the Command Palette and
 point to the .vsix file.
 
-See below for installing an extension from the cli.
+You can install an extension on the CLI with:
+
+```bash
+# From the Coder extension marketplace
+code-server --install-extension ms-python.python
```

Comment:
Will this still work once we switch to Open-VSX?

Reply:
Yep! It's on the store - https://open-vsx.org/extension/ms-python/python - I'll change the comment to reflect it.